app/models/organisation_members.py

```diff
@@ -11,5 +12,5 @@ class OrganisationMembers(ModelMixin):
     _tablename_ = "organisation_members"
 
     id = db.Column(db.Integer, primary_key=True)
```

Comment:
```suggestion
   id = db.Column(UUID(as_uuid=True), primary_key=True, server_default=sa_text("uuid_generate_v4()"))
```

Reply:
We can keep ids of the associate tables as integers, they are more performant that and they are not a security risk.